src/lib/cli.rs

```diff
@@ -47,6 +50,8 @@ fn run(cli_args: CliArgs, global_config: &GlobalConfig) {
         version::check();
     }
 
+    recursion_level::increase_level();
```

Comment:
i think this needs to be set at first line of the run. before any invocation of the logger.
otherwise it means you have 2 different levels at same cargo-make process...